src/main/java/school/lemon/changerequest/java/introduction/hw1/Task6.java

```diff
@@ -2,10 +2,10 @@
 
 public class Task6 {
     public static double calculateS(double x) {
-        return 0d;
+        return (1+x+Math.pow(x,2)/2+Math.pow(x,3)/6+Math.pow(x,4)/24);
```

Comment:
Let's say you wanna do the same calculations but 10 times instead of 4. Are you still going to copy-paste?